environments/environments.py

```diff
@@ -54,11 +54,11 @@ def _draw(self):
         """
         raise NotImplementedError
 
-    def _sample_init_conditions(self, radius):
+    def _sample_init_conditions(self, radius_bound):
         """Samples random initial conditions for the environment
 
         Args:
-            radius (float): Radius of the sampling process
+            radius_bound (float, float): Radius lower and upper bound of the phase state sampling.
```

Comment:
Since `radius_bound` is a tuple, should the docstring be something like
```suggestion
            radius_bound (tuple): Tuple with radius lower and upper bound of the phase state sampling.
```

Reply:
Another question: why is the name radius? Can it be something more general?

Reply:
it is radius because in the paper they call it radius. You can find the information in section 4.

Reply:
Ok sorry I didn't remember that